core/domain/email_domain.py

```diff
@@ -70,3 +70,37 @@ def thread_id(self):
         """
         return '.'.join(
             [self.entity_type, self.entity_id, self.id.split('.')[3]])
+
+
+class SendEmailInfo(python_utils.OBJECT):
+    """The information needed to send an email.
+    Attributes:
+        recipient_id: str. The user ID of the email recipient.
+        sender_id: str. The user ID of the sender.
+        intent: str. The intent string for the email, i.e. the purpose/type.
```

Comment:
Suggest mentioning where in feconf these are defined so that the caller may be aware of which value are accepted.